Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/worker/WorkerBinder.kt

```diff
@@ -71,4 +72,11 @@ abstract class WorkerBinder {
     abstract fun deadmanNotificationPeriodic(
         factory: DeadmanNotificationPeriodicWorker.Factory
     ): InjectedWorkerFactory<out ListenableWorker>
+
+    @Binds
+    @IntoMap
+    @WorkerKey(ContactDiaryCleanWorker::class)
+    abstract fun contactDiaryCleanWorker(
```

Comment:
Should we create a contact diary specific worker module?
We could also create a ContactDiaryRentionModule.

Ideally we keep the diary separate such that in the future if we want to go multi module, it would be easier.